### ISSUE TITLE
webgl support

### ISSUE THREAD

**Issue by mattcolman** [raksa/phaser-dragonbones#1]
Hi, from my testing this only works with the Canvas renderer. How come?

Thanks,

**Comment by raksa** [comment]
Yep, I'm seeking somebody expert on Phaser's [Rope](http://phaser.io/examples/v2/sprites/rope) to implement it too.

**Comment by mattcolman** [comment]
ok, I gather it only needs to use Rope for mesh transforms right?

**Comment by raksa** [comment]
Yes Phaser has Rope as its feature for mesh rendering.

**Comment by kheftel-old** [comment]
Might be a good idea to mention that WebGL isn't supported in the readme. I guess I know now what the cryptic "This plugin can't support kind of mesh rendering yet" text means ;)
